src/Bang.Analyzers.Tests/Bang.Analyzers.Tests.csproj

```diff
@@ -0,0 +1,32 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFramework>net7.0</TargetFramework>
+        <ImplicitUsings>enable</ImplicitUsings>
+        <Nullable>enable</Nullable>
+        <IsPackable>false</IsPackable>
+        <AssemblyName>Bang.Analyzers.Tests</AssemblyName>
+        <AssemblyVersion>1.0.0.0</AssemblyVersion>
+    </PropertyGroup>
+
+    <ItemGroup>
+        <PackageReference Include="Microsoft.CodeAnalysis.CSharp.Analyzer.Testing.XUnit" Version="1.1.1" />
+        <PackageReference Include="Microsoft.CodeAnalysis.CSharp.CodeFix.Testing" Version="1.1.1" />
+        <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.3.2" />
+        <PackageReference Include="xunit" Version="2.4.2" />
```

Comment:
Yess!! Tests!!! :D
One question though: why did you add xunit and Microsoft.NET.Sdk? I was using just `Microsoft.NET.Test.Sdk` for Gum and it was working all right.

Reply:
TBH this is just the default Rider template for unit tests and xUnit is my favourite test framework 😂 

I'm happy to use whatever you are more comfortable with, so I can drop xUnit if you so wish